app/src/main/java/org/mozilla/focus/menu/browser/BrowserMenuAdapter.kt

```diff
@@ -111,7 +111,8 @@ class BrowserMenuAdapter(
                     R.drawable.ic_settings_menu_item))
         }
 
-        if (AppConstants.isGeckoBuild(context)) {
+        if (AppConstants.isGeckoBuild) {
+            items.add(MenuItem.RequestDesktopCheck)
```

Comment:
Oh, how did this item get in here? Is this a related change?

Reply:
I'm fairly certain it came from a merge conflict. I can pull it.